book/09-git-and-other-scms/sections/import-bzr.asc

```diff
@@ -0,0 +1,154 @@
+==== Bazaar
+(((Bazaar)))(((Importing, from Bazaar)))
+
+Bazaar is a distributed source management content like Git.
+It is very easy to convert a Bazaar repository into a Git repository thanks to Bazaar's plugin system.
+To convert a Bazaar repository into a Git repository, you have first to import the bzr-fastimport plugin.
+Here is the whole procedure if you have not yet imported any plugin:
```

Comment:
I added some instructions to install bzr-fastimport since I realized that it is available in the Fedora and Debian repositories. I first suggest to install it using the usual command, quoting some of the main Linux distributions (I put the examples using apt-get, yum and dnf).